### ISSUE TITLE
Dataset(s) listed in multiple repositories

### ISSUE THREAD

**Issue by tommyclay** [FlukeAndFeather/openbiologging#4]
Another Q - If the same dataset is provided through two repositories/links, do we add them both as separate rows in the data tab, or just input that which adheres most to FAIR principles? From the Cleasby et al. 2015 example, GPS data are provided through a Seabird Tracking Database ID (though the ID is for the data holder and not the dataset and the URL doesn’t work, so the dataset is not findable) and a Dryad link (along with the pressure and temperature data). Is it best to enter information separately for the seabird tracking database and dryad repositories, or just Dryad?

**Comment by FlukeAndFeather** [comment]
For the broad pass, we're just going to record whether or not the data have a data availability statement. This example passes that test. However we'll need to think about this for the fine-scale pass. I think we should decide a general rule, either pick the most FAIR-compliant repository (Dryad in this case, assuming the tracking data are in there) or pick the domain-relevant repository (Seabird Tracking Database).

**Comment by tommyclay** [comment]
I agree, it'd be interesting to discuss this more. I'm leaning towards picking the most FAIR-compliant, but it's a useful to note that since BirdLife updated their Seabird Tracking Database (I think last year), many of the urls are now broken. This emphasizes the importance of a dataset with a distinct DOI.
